aiida_mlip/helpers/help_load.py

```diff
@@ -40,7 +40,11 @@ def load_model(
         The loaded model.
     """
     if model is None:
-        loaded_model = None
+        loaded_model = ModelData.download(
```

Comment:
I think the docstrings (longer function description and `model` description) should mention this behaviour.